docker/hq-compose-formplayer.yml

```diff
@@ -0,0 +1,23 @@
+formplayer:
+  extends:
+    file: hq-compose.yml
+    service: formplayer
+  environment:
+    DEPENDENT_SERVICES: "WEB:8000 COUCH:5984 POSTGRES:5432"
```

Comment:
I don't think listing `WEB:8000` here is going to work unless you also list it under `links` since this relies on docker environment variables.